insights/parsers/tests/test_du_usage.py

```diff
@@ -0,0 +1,50 @@
+from insights.parsers.du_usage import DBDiskSpace
+from insights.tests import context_wrap
+
+dbdisk_content = """
+COMMAND> du -sh /var/lib/mongodb
+
+22G     /var/lib/mongodb
```

Comment:
Please refer my above comment regarding showing list of paths.

Reply:
done